purchase_sale_link_by_origin/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,5 @@
+By default since `Odoo 14.0 <https://github.com/odoo/odoo/commit/5a1645a8f8f3560eb778da90b6160b322ce0722e#diff-867f19b2ba78aa5cfaeeb601d4a04cbcbe64e389a8d38939be06c1e547d2934a>`_, PO and SO are linked by their order lines.
```

Comment:
```suggestion
By default since `Odoo 14.0 <https://github.com/odoo/odoo/commit/5a1645a8f8f3560eb778da90b6160b322ce0722e>`_, PO and SO are linked by their order lines.
```